src/AddressFormat/AddressFormat.php

```diff
@@ -372,6 +379,21 @@ public function getPostalCodePrefix()
         return $this->postalCodePrefix;
     }
 
+    /**
+     * Gets the postal code example.
+     *
+     * This is an example postal code used to provide additional context to the
+     * end-user. It can be displayed as a placeholder, field description or as
+     * part of the validation.
+     * (E.g. Zip code 12345 is not valid for Louisana (Example: 70001).
```

Comment:
Sentence starts with a parenthesis, but doesn't end with one. I suggest just leaving out the first one and starting with just "E.g."

Reply:
Thanks for the feedback. I've addressed this in https://github.com/commerceguys/addressing/pull/128/commits/7cb62ca80a0279ecb8d552302850f13a2d8388a1